src/java.base/share/classes/jdk/internal/foreign/SegmentBulkOperations.java

```diff
@@ -202,9 +202,9 @@ public static int contentHash(AbstractMemorySegmentImpl segment, long fromOffset
             return 1;
         }
         int result = 1;
-        final long longBytes = length & ((1L << 62) - 8);
+        final long longBytes = length & LONG_MASK;
```

Comment:
Isn't `LONG_MASK` different from the old constant?

Reply:
The difference is that bit 62 and 63 are now `1` rather than `0`. This will not have any significant implications.